src/p2p/net_node.inl

```diff
@@ -2919,8 +2919,8 @@ namespace nodetool
     const uint32_t index = stripe - 1;
     CRITICAL_REGION_LOCAL(m_used_stripe_peers_mutex);
     MINFO("adding stripe " << stripe << " peer: " << context.m_remote_address.str());
-    std::remove_if(m_used_stripe_peers[index].begin(), m_used_stripe_peers[index].end(),
-        [&context](const epee::net_utils::network_address &na){ return context.m_remote_address == na; });
+    m_used_stripe_peers[index].erase(std::remove_if(m_used_stripe_peers[index].begin(), m_used_stripe_peers[index].end(),
+        [&context](const epee::net_utils::network_address &na){ return context.m_remote_address == na; }), m_used_stripe_peers[index].end());
```

Comment:
The code looks like a valid applucation of the Erase-Remove idiom, but I would prefer, that you stored the result of `m_used_stripe_peers[index]` as a non-const referece `peer` and reuse it, for much better readability.

Reply:
This would be an unrelated change to this bugfix. Besides, indexing an `std::array` gives almost 0 overhead after compiler optimizations.

Reply:
Seeing that this is already merged to master no point in changing anymore.

Reply:
> This would be an unrelated change to this bugfix

I see it a bit differently. It is a chance to improve readability since you are already touching a given line. Perhaps others would disagree.

> Besides, indexing an std::array gives almost 0 overhead after compiler optimizations.

As I said, it's not about speed, but readability. Even without optimisations the gain w.r.t. the remaining code would be negligible.

Plus, this is not a request to change anymore. Just my hint for future.